src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  margin: auto 0;
+  min-width: 1024px;
+  max-width: 1200px;
```

Comment:
no need to limit width. Let's make it work fine for all screens
![image](https://github.com/mate-academy/layout_moyo-header/assets/98764641/65d5ad8b-8cc0-4eac-88c3-55e0e904a36c)